core/src/autogluon/core/_setup_utils.py

```diff
@@ -26,9 +26,9 @@
     "networkx": ">=3.0,<4",  # Major version cap
     "tqdm": ">=4.38,<5",  # Major version cap
     "Pillow": ">=10.0.1,<11",  # Major version cap
-    "torch": ">=2.0,<2.2",  # "<{N+2}" upper cap, sync with common/src/autogluon/common/utils/try_import.py
-    "lightning": ">=2.0.0,<2.2",  # "<{N+2}" upper cap
-    "pytorch_lightning": ">=2.0.0,<2.2",  # "<{N+2}" upper cap, capping `lightning` does not cap `pytorch_lightning`!
+    "torch": ">=2.1,<2.2",  # "<{N+2}" upper cap, sync with common/src/autogluon/common/utils/try_import.py
```

Comment:
are we taking off support for Torch 2.0 completely ?

Reply:
Yes, it would install 2.1 anyway. Looks like no need to keep 2.0 here.